doc/source/release-notes/Percona-Server-8.0.28-19.rst

```diff
@@ -0,0 +1,56 @@
+.. _PS-8.0.28-19:
+
+================================================================================
+*Percona Server for MySQL* 8.0.28-19
+================================================================================
+
+:Date: March 2022
+:Installation: `Installing Percona Server for MySQL <https://www.percona.com/doc/percona-server/8.0/installation.html>`_
+
+`Percona Server for MySQL <https://www.percona.com/software/mysql-database/percona-server>`_ 8.0.28-19
+includes all the features and bug fixes available in the
+`MySQL 8.0.28 Community Edition <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-28.html>`__.
+in addition to enterprise-grade features developed by Percona.
+
+*Percona Server for MySQL* is a free, fully compatible, enhanced and open
+source drop-in replacement for any *MySQL* database. It provides superior
+performance, scalability, and instrumentation.
+
+*Percona Server for MySQL* is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads, and
+delivers greater value to *MySQL* server users with optimized performance,
+greater performance scalability and availability, enhanced backups and
+increased visibility. `Commercial support contracts are available
+<https://www.percona.com/services/support/mysql-support>`__.
+
+.. contents::
+   :local:
+
+Release Highlights
+=================================================
+
+The following lists a number of the bug fixes for *MySQL* 8.0.28, provided by Oracle, and included in Percona Server for MySQL:
+
+* Using the ``ASCII`` shortcut for ``CHARACTER SET latin1`` and ``UNICODE`` shortcut for ``CHARACTER SET ucs2`` are deprecated. Using either shortcut raises a warning to use ``CHARACTER SET`` instead. Expect that these shortcuts will be removed in a future version.
+* A stored function and a loadable function with the same name can share the same namespace. Add the schema name when invoking a stored function in the shared namespace. The server generates a warning when function names collide.
+* InnoDB supports ``ALTER TABLE ... RENAME COLUMN`` operations when using ``ALGORITH=INSTANT``. 
+* The limit for ``innodb_open_files`` includes temporary tablespace files. This addition is a change and the temporary tablespace files were not counted in previous versions.
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.28 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-28.html>`__.
+
+Bugs Fixed
+=================================================
+
+* :jirabug:`PS-7999` The ``FEDERATED`` storage engine would not reconnect when a ``wait_timeout`` was exceeded. (Thanks to Sami Ahlroos for reporting this issue) (Upstream :mysqlbug:`105878`)
```

Comment:
:jirabug:`PS-7999` -> add a colon after the ticket number `PS-7999`:

Reply:
added